apps/demos/Demos/Menu/Scrolling/React/App.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback, useState } from 'react';
+import Menu, { MenuTypes } from 'devextreme-react/menu';
+import CheckBox, { CheckBoxTypes } from 'devextreme-react/check-box';
+import notify from 'devextreme/ui/notify';
+import service from './data.ts';
+
+const SUBMENU_HEIGHT = 200;
+const products = service.getProducts();
+
+const App = () => {
+  const [limitSubmenuHeight, setLimitSubmenuHeight] = useState(false);
+
+  const itemClick = useCallback((e: MenuTypes.ItemClickEvent) => {
+    if (!e.itemData.items) {
+      notify(`The "${e.itemData.text}" item was clicked`, 'success', 1500);
+    }
+  }, []);
+
+  const limitSubmenuHeightOnMouseClick = useCallback((e: CheckBoxTypes.ValueChangedEvent) => {
+    setLimitSubmenuHeight(e.value);
+  }, [setLimitSubmenuHeight]);
+
+  const onSubmenuShowing = useCallback(({ submenuContainer }: any) => {
```

Comment:
not sure, probably this is of type HTMLElement, plz check if we can get rid of any

Reply:
i meant by specifying HTMLElement type, not by removing it )